advanced/Scripts/piholeDebug.sh

```diff
@@ -151,6 +160,17 @@ function hostnameCheck {
     fi
 }
 
+function portCheck {
+    echo "############################################################" >> ${DEBUG_LOG}
+	echo "########           Open Port Information           #########" >> ${DEBUG_LOG}
+	echo "############################################################" >> ${DEBUG_LOG}
+
+    echo "::: Writing local server ports to logfile"
+
+    ${SUDO} netstat -tulpn >> ${DEBUG_LOG}
```

Comment:
correction, it wasn't that netstat wasn't installed it was that it was that alpine's busybox version which had quite different functionality and flags than the debian version.  LSOF seems to be same between alpine/debian.